tests/baselines/reference/user/lodash.log

```
@@ -126,8 +126,6 @@ node_modules/lodash/_createHybrid.js(64,15): error TS2454: Variable 'holdersCoun
 node_modules/lodash/_createHybrid.js(65,31): error TS2532: Object is possibly 'undefined'.
 node_modules/lodash/_createHybrid.js(68,9): error TS2345: Argument of type 'TimerHandler' is not assignable to parameter of type 'Function'.
   Type 'string' is not assignable to type 'Function'.
-node_modules/lodash/_createHybrid.js(68,46): error TS2339: Property 'placeholder' does not exist on type '(...args: any[]) => any'.
```

same -- subsequent errors are not reported